### ISSUE TITLE
Have scan oscillation overrides changed to use radians?

### ISSUE THREAD

**Issue by dagewa** [dials/dials#2606]
Feel like I'm going crazy... On my up to date build

```
$ dials.import $(dials.data get -q x4wide)/X4_wide_M1S4_2_0001.cbf geometry.scan.oscillation=1,0.5
$ dials.show imported.expt | grep oscillation
```
gives
```
    oscillation:   {1,0.00872665}
```
but on DIALS 3.17.0, the same procedure gives
```
    oscillation:   {1,0.5}
```

**Comment by dagewa** [comment]
Actually it's not only overrides. Importing normally then printing the oscillation gives the second value in radians:
```
$ dials.import $(dials.data get -q x4wide)/X4_wide_M1S4_2_0001.cbf
$ dials.show imported.expt | grep oscillation
```
```
    oscillation:   {0,0.0174533}
```

**Comment by toastisme** [comment]
Fixed as part of [#693](https://github.com/cctbx/dxtbx/pull/693)
